tests/Http/Offer/AddLabelFromJsonBodyRequestHandlerTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Offer;
+
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Http\Response\AssertJsonResponseTrait;
+use CultuurNet\UDB3\Http\Response\NoContentResponse;
+use CultuurNet\UDB3\LabelJSONDeserializer;
+use CultuurNet\UDB3\Model\ValueObject\Taxonomy\Label\Label;
+use CultuurNet\UDB3\Model\ValueObject\Taxonomy\Label\LabelName;
+use CultuurNet\UDB3\Offer\Commands\AbstractLabelCommand;
+use CultuurNet\UDB3\Offer\Commands\AddLabel;
+use PHPUnit\Framework\TestCase;
+
+final class AddLabelFromJsonBodyRequestHandlerTest extends TestCase
+{
+    use AssertApiProblemTrait;
+    use AssertJsonResponseTrait;
+
+    private const OFFER_ID = 'd2a039e9-f4d6-4080-ae33-a106b5d3d47b';
+    private const LABEL_NAME = 'Some new Label';
+
+    private TraceableCommandBus $commandBus;
+
+    private AddLabelFromJsonBodyRequestHandler $addLabelFromJsonBodyRequestHandler;
+
+    private Psr7RequestBuilder $psr7RequestBuilder;
+
+    protected function setUp(): void
+    {
+        $this->commandBus = new TraceableCommandBus();
+
+        $this->addLabelFromJsonBodyRequestHandler = new AddLabelFromJsonBodyRequestHandler(
+            $this->commandBus,
+            new LabelJSONDeserializer()
+        );
+
+        $this->psr7RequestBuilder = new Psr7RequestBuilder();
+
+        $this->commandBus->record();
+    }
+
+    /**
+     * @test
+     * @dataProvider offerTypeDataProvider
+     */
+    public function it_handles_adding_a_label_to_an_offer(
+        string $offerType,
+        string $type,
+        AbstractLabelCommand $addLabel
+    ): void {
+        $addLabelFromJsonBodyRequest = $this->psr7RequestBuilder
+            ->withRouteParameter('offerType', $offerType)
+            ->withRouteParameter('offerId', self::OFFER_ID)
+            ->withJsonBodyFromArray([
+                'label' => self::LABEL_NAME,
+                'offers' => [
```

Comment:
I don't think this is property `offers` is needed.

Reply:
When possible I've looked at the old tests => https://github.com/cultuurnet/udb3-silex/blob/11df7ea144d44070f568bc7f429e3630e5775df7/tests/Http/Offer/EditOfferRestControllerTest.php#L114 
It seemed prudent, to test the same body.
But looking at `LabelJSONDeserializer` it indeed seems useless.